src/utils/replicate/mixtral_api.ts

```diff
@@ -1,7 +1,22 @@
-import type { CacheType, CommandInteraction } from 'discord.js';
+import { partition_text } from '@utils/text';
+import type { CacheType, CommandInteraction, Message } from 'discord.js';
 
 import { REPLICATE } from '.';
 
+const MODEL = 'mistralai/mixtral-8x7b-instruct-v0.1';
+
+const MAX_MESSAGE_LENGTH = 2000;
+
+export const DEFAULT_MIXTRAL_VALUES = {
+    max_new_tokens: 512,
+    temperature: 0.6,
+    top_p: 0.9,
+    top_k: 50,
+    presence_penalty: 0,
+    frequency_penalty: 0,
+    prompt_template: '<s>[INST] {prompt} [/INST] ',
+};
+
 export interface InputType {
```

Comment:
you can also do something like
```ts
type Input = typeof DEFAULT_MIXTRAL_VALUES;
```
that way you are sure that the default values represents the input type.
else you'd probably want to annotate `DEFAULT_MIXTRAL_VALUES` with its type to be sure it matches it (in case of future changes)